ReactiveCocoa/Swift/Action.swift

```diff
@@ -134,7 +134,7 @@ public final class Action<Input, Output, Error: ErrorType> {
 public final class CocoaAction: NSObject {
 	/// The selector that a caller should invoke upon a CocoaAction in order to
 	/// execute it.
-	public static let selector: Selector = "execute:"
+	public static let selector: Selector = #selector(CocoaAction.execute(_:))
```

Comment:
Sorry for the delay… I was traveling all day yesterday (with the whole family) and am just catching up on things.

I'm hesitant to give up on semver, but given the way Swift/Xcode work it's not very practical, as you mentioned. I can be okay with breaking changes in minor RAC versions for minor Swift versions. 

But I'd **really** like us to make releases like RAC 4.1 compatible with both Swift 2.1 and Swift 2.2 if possible. Then we could make RAC 4.2 compatible with Swift 2.2 only. Upgrading Xcode versions is a major hassle because _everything breaks_. If we can be compatible with both for a single minor release, I think that will significantly ease the pain for our consumers. (Benefitting both the frameworks and the apps that I maintain.)

But ultimately, you're the one doing the work, so it depends on what you're willing to do.